lstchain/reco/tests/test_utils.py

```diff
@@ -209,21 +221,28 @@ def test_get_geomagnetic_delta():
 
     # this is just a regression test assuming the values were correct when
     # first implementing the function
-    inc = get_geomagnetic_delta(zen=20 * u.deg, az=0 * u.deg, time=GEOM_MAG_REFERENCE_TIME)
+    inc = get_geomagnetic_delta(
+        zen=20 * u.deg, az=0 * u.deg, time=GEOM_MAG_REFERENCE_TIME
+    )
     assert u.isclose(inc, 1.26667967 * u.rad)
 
-    inc = get_geomagnetic_delta(zen=50 * u.deg, az=20 * u.deg, time=GEOM_MAG_REFERENCE_TIME)
+    inc = get_geomagnetic_delta(
+        zen=50 * u.deg, az=20 * u.deg, time=GEOM_MAG_REFERENCE_TIME
+    )
     assert u.isclose(inc, 1.73389012 * u.rad)
 
 
 def test_apply_src_r_cut(simulated_dl1_file):
     from lstchain.io.io import dl1_params_lstcam_key
     from lstchain.reco.utils import apply_src_r_cut
     from lstchain.io.config import get_srcdep_config
-    
+
     params = pd.read_hdf(simulated_dl1_file, key=dl1_params_lstcam_key)
     srcdep_config = get_srcdep_config()
-    src_r_min = srcdep_config['train_gamma_src_r_deg'][0]
-    src_r_max = srcdep_config['train_gamma_src_r_deg'][1]
+
+    src_r_min = srcdep_config["train_gamma_src_r_deg"][0]
+    src_r_max = srcdep_config["train_gamma_src_r_deg"][1]
+
     params = apply_src_r_cut(params, src_r_min, src_r_max)
-    assert (params.event_id.values == np.array([998705, 1680619])).all()
+
+    assert (params.event_id.values == np.arange(100, 110, 1)).all()
```

Comment:
This is the only difference in the actual code. Sorry for mixing in with the formatting edit.